src/NuGet.Clients/PackageManagement.UI/Xamls/PackageManagerTopPanel.xaml.cs

```diff
@@ -200,4 +201,17 @@ public void SelectFilter(Filter selectedFilter)
             _selectedFilter.Selected = true;
         }
     }
+
+    public class FilterChangedEventArgs : EventArgs
+    {
+        public Filter? PreviousFilter
+        {
+            get;
+        }
+
+        public FilterChangedEventArgs(Filter? previousFilter)
```

Comment:
This confused me a bit, we could have used a Filter.None instead, but I guess this works and might actually be cleaner.